### DISCUSSION TITLE
求助：使用nonebot连接mirai2后某些插件用不了

### DISCUSSION THREAD

**Discussion by yangjiaji1795** [nonebot/discussions#437] [问答]
![屏幕截图 2024-02-13 145358](https://github.com/nonebot/discussions/assets/73827879/7e349a3f-0c0f-4431-a93e-9bd5b05236f2)
可以看到nonebot_plugin_oddtext插件可以使用而echo插件用不了。

**Comment by yanyongyu** [comment]
echo插件在群里需要at机器人才会回复

**Reply by yangjiaji1795** [reply]
那是不是其他有些插件用不了，我试了其他的不行。
看之前的Discussions说有些插件不支持。

**Reply by yanyongyu** [reply]
你可以前往 https://registry.nonebot.dev/ 查看插件支持的适配器

**Reply by yangjiaji1795** [reply]
感谢！
